runtime/rust/src/func/attach.rs

```diff
@@ -32,7 +32,7 @@ pub fn wasm_attach_bpf_program(
     };
     if let Some(attach_target) = attach_target_str {
         let section_name = program.section();
-        // TODO: support more attach type
```

Comment:
please add this as an issue:

maybe add more examples on more bpf program types?

Reply:
https://github.com/eunomia-bpf/wasm-bpf/issues/35